.github/boring-cyborg.yml

```diff
@@ -280,9 +280,11 @@ labelPRBasedOnFilePath:
         - extensions/vertx/**/*
         - extensions/vertx-core/**/*
         - extensions/vertx-graphql/**/*
+        - extensions/vertx-redis/**/*
         - extensions/vertx-http/**/*
         - extensions/vertx-keycloak/**/*
         - extensions/vertx-web/**/*
         - integration-tests/vertx/**/*
         - integration-tests/vertx-graphql/**/*
+        - integration-tests/vertx-redis/**/*
```

Comment:
I would probably add a specific label for it.

Reply:
Sure, I am adding the `area/redis` label. We'll need it created I suppose.